### PR TITLE
https://code4africa.slack.com/archives/G0DNN0CN5/p1678696313660149

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

Makes content localized.

Fixes # [issue](https://code4africa.slack.com/archives/G0DNN0CN5/p1678696313660149)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/charterafrica/src/lib/payload/index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ async function findPage(slug, options) {
 
 async function getCollection(collection, options) {
   return payload.find({
+    limit: 0,
```
Comment (kilemensi) [inline apps/charterafrica/src/lib/payload/index.js:17]: What is the default limit?

**Review by kilemensi — APPROVED** [review]
🚀

File: apps/charterafrica/src/lib/payload/index.js (hunk 1 above)
Comment (koechkevin) [reply apps/charterafrica/src/lib/payload/index.js:17]: Defaults is 10

Comment (kilemensi) [reply apps/charterafrica/src/lib/payload/index.js:17]: I take it 0 means no limit then?

Comment (koechkevin) [reply apps/charterafrica/src/lib/payload/index.js:17]: Yes

**Pull request merged**